examples/soap/mensagens_do_dia.py

```diff
@@ -0,0 +1,32 @@
+import os
+
+from as_ws_wrapper.wrapper.as_ws import AccesstageSoapWrapper
+
+from examples.utils import dump_response, read_json
+
+try:
+    mensagens_data = read_json(os.path.basename(__file__).split('.')[0])
+except Exception:
+    mensagens_data = []
+
+mensagens_data_trkIdIn = [
+    msg['trkIdIn'] for msg in mensagens_data
+]
+
+r = AccesstageSoapWrapper().lista_mensagens('1628267')
+
+mensagens = []
+
+for msg in r.data:
+    if msg['tipoDocumento'] in ['PAGAMENTO', 'OUTOS '] and msg['trkIdIn'] not in mensagens_data_trkIdIn:
```

Comment:
Erro de digitação copy/paste?
```suggestion
    if msg['tipoDocumento'] in ['PAGAMENTO', 'OUTROS '] and msg['trkIdIn'] not in mensagens_data_trkIdIn:
```

Reply:
Talvez, vou buscar novamente